src/components/TransactionTable/TransactionTable.js

```diff
@@ -0,0 +1,214 @@
+import React, { PureComponent } from 'react';
+import propTypes from 'prop-types';
+import moment from 'moment';
+import { Table } from '../index';
+import { connectTheme } from '../../utils';
+
+import TransactionView from './TransactionView';
+
+const constants = {
+  //DATE_FORMAT: 'YYYY-MM-DD hh:mm a',
+  DATE_FORMAT: 'MM/DD/YY hh:mm a',
+  SEND: 'Sent',
+  RECEIVE: 'Received',
+  COINBASE_RECEIVE: 'Coinbase',
+  MULTIPLE_OUTPUT_COPY: 'Multiple',
+  UNKNOWN_ADDRESS_COPY: 'Unknown',
```

Comment:
Something that has been on the back of my mind for a while now, and this only made me re-think about it, is preparing for internationalization so that things can be translated... Not sure if you have any experience with good tools or strategies for dealing with that but thought I'd bring it up.